### PR TITLE
10741 address region fix

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#10741

*Description of changes:*
- force the app to register the region rules change

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]

File: ppr-ui/src/composables/address/BaseAddress.vue [hunk 1]
```diff
@@ -210,7 +208,10 @@ export default defineComponent({
         schemaLocal.value.region = origRegionRules
       } else {
         schemaLocal.value.postalCode = origPostalCodeRules.concat([baseRules.maxLength(15)])
-        schemaLocal.value.region = [baseRules.maxLength(2), ...spaceRules]
+        for (let i = 0; i < schemaLocal.value.region.length; i++) {
```
Comment (kialj876) [inline ppr-ui/src/composables/address/BaseAddress.vue:211]: hey steph, this should be `schemaLocal.value.region.length - 1` I think? Might not matter depending on how the array pop() works on an empty list

**Review by kialj876 — APPROVED** [review]
Just one question, but looks good

File: ppr-ui/src/composables/address/BaseAddress.vue (hunk 1 above)
Comment (jordiwes) [reply ppr-ui/src/composables/address/BaseAddress.vue:211]: if the length is 3, it's going to pop 0, 1, and 2 , which are 3 entries.

I did discover that the concat wasn't behaving nicely though, so I made a change there

Comment (kialj876) [reply ppr-ui/src/composables/address/BaseAddress.vue:211]: Oh right yeah definitely haha. Uhhh but without the concat we're back to the same thing as before no?

Comment (kialj876) [reply ppr-ui/src/composables/address/BaseAddress.vue:211]: I guess if it works it works. Maybe since it does the 'pops' it triggers the change this way

**Review by kialj876 — APPROVED** [review]

**Pull request merged**